### PR TITLE
Require Signature Verification in Randao, Attestations and AttesterSlashings

### PR DESCRIPTION
**Pull request by 0xKiwi**
This PR is part of #2951 

This PR removes the verifySignature flag from the processes for RandaoReveal, Attestations, and AttesterSlashings. Since this means all of the operations now require BLS verification, the config parameter has been removed from ProcessOperations 😄

### CODE REVIEW

File: beacon-chain/rpc/proposer_server_test.go [hunk 1]
```diff
@@ -2,11 +2,14 @@ package rpc
...
 import (
 	"context"
+	"crypto/rand"
 	"math/big"
 	"reflect"
 	"strings"
 	"testing"
 
+	"github.com/prysmaticlabs/prysm/shared/bls"
+
```
Comment (prestonvanloon) [inline beacon-chain/rpc/proposer_server_test.go]: Imports

File: beacon-chain/core/state/transition.go [hunk 2]
```diff
@@ -232,7 +231,7 @@ func ProcessOperations(
 	ctx context.Context,
 	state *pb.BeaconState,
 	body *ethpb.BeaconBlockBody,
-	config *TransitionConfig) (*pb.BeaconState, error) {
```
Comment (prestonvanloon) [inline beacon-chain/core/state/transition.go]: Please delete the entire struct TransitionConfig

Comment (0xKiwi) [reply beacon-chain/core/state/transition.go]: I would but verifyStateRoot still requires it

Comment (prestonvanloon) [reply beacon-chain/core/state/transition.go]: Discussed offline that this will be resolved in @terencechain upcoming PR.

**Pull request merged**